appintro/src/main/java/com/github/paolorotolo/appintro/AppIntro2.kt

```diff
@@ -33,6 +46,11 @@ abstract class AppIntro2 : AppIntroBase() {
 
     override fun getLayoutId() = R.layout.appintro_intro_layout2
 
+    /**
+     * Returns the backgroundFrame to use with image loading libraries like Glide
+     */
+    fun getBackgroundFrame() = backgroundFrame
```

Comment:
Why this? I would advice we don't do that honestly.

Reply:
Ummm. Why?

Reply:
Done